api/tests/test_status.py

```diff
@@ -21,8 +21,9 @@ def setUp(self):
     def testCreate(self):
         self.create(self.data_dict)
 
-    def testList(self):
-        self.list()
+    def testList(self):     
+        response = self.client.get(reverse(self.prefix + "-list", kwargs={"team_pk": self.team.id}), format="json")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
I think rather than removing the base function call it would work better to add a parameter to `self.list` to ignore page size, **_or_** take use of the expected keyword and already make your own query set.

Reply:
Might be good to accept an optional pagination class, and then pass the associated class variable where necessary.